src/assets/styles/pages/_resources.scss

```diff
@@ -0,0 +1,388 @@
+        content: '\2713';
+        display: inline-block;
+        font-size: rem(18);
+        height: rem(2);
+        left: rem(4);
+        position: absolute;
+        top: 0;
+        width: rem(2);
+      }
+    }
+
+    // End of custom checkboxes
+  }
+
+  .apply-button {
+    background-color: var(--fl-fgColor, var(--indigo-500));
+    border: 2px solid var(--fl-fgColor, var(--indigo-500));
+    border-radius: rem(5);
+    color: var(--fl-bgColor, $white);
+    font-size: rem(16);
+    font-weight: var(--fw-medium);
+    line-height: rem(24);
+    margin-top: rem(16);
+    padding: rem(10) rem(14);
+
+    &:hover {
+      background-color: var(--fl-fgColor, var(--indigo-800)) !important;
+      border: 2px solid var(--fl-fgColor, var(--indigo-800)) !important;
+      color: var(--fl-bgColor, #fff) !important;
+    }
+
+    &:focus {
+      background-color: var(--fl-fgColor, var(--indigo-800));
+      border: 2px solid var(--fl-fgColor, var(--indigo-800));
+      color: var(--fl-bgColor, #fff);
+      outline: var(--fl-fgColor, var(--indigo-500)) solid rem(3);
+      outline-offset: rem(3);
+    }
+  }
+
+  .reset-button {
+    background-color: var(--fl-fgColor, var(--grey-100));
+    border: 2px solid var(--fl-fgColor, var(--indigo-500));
+    border-radius: rem(5);
+    color: var(--fl-bgColor, var(--indigo-500));
+    font-size: rem(16);
+    font-weight: var(--fw-medium);
+    padding: rem(10) rem(14);
+
+    &:hover {
+      background-color: var(--fl-fgColor, var(--indigo-800)) !important;
+      border: 2px solid var(--fl-fgColor, var(--indigo-800)) !important;
+      color: var(--fl-bgColor, #fff) !important;
+    }
+
+    &:focus {
+      background-color: var(--fl-fgColor, var(--indigo-800));
+      border: 2px solid var(--fl-fgColor, var(--indigo-800));
+      color: var(--fl-bgColor, #fff);
+      outline: var(--fl-fgColor, var(--indigo-500)) solid rem(3);
+      outline-offset: rem(3);
+    }
+  }
+
+  .resources-no-results {
+    background-color: #f2f2f2;
+    font-size: 22px;
+    font-weight: 600;
+    padding: rem(16);
+    padding-left: rem(20);
+  }
+
+  .resources-applied-filters {
+    margin: 0;
+    padding-bottom: rem(16);
+
+    h3 {
+      margin-top: 0;
+    }
+
+    .filter-tags {
+      display: flex;
+      flex-wrap: wrap;
+      gap: rem(16);
+
+      .filter-tag {
+        background-color: #f2f2f2;
+        border: 2px solid var(--fl-fgColor, var(--indigo-500));
+        border-radius: rem(48);
+        column-gap: rem(4);
+        display: flex;
+        justify-content: center;
+        padding: rem(8);
+        white-space: nowrap;
+
+        p {
+          color: var(--fl-fgColor, var(--indigo-500));
+          margin: auto;
+        }
+
+        svg {
+          height: rem(16);
+          margin: auto;
+          stroke: var(--fl-fgColor, var(--indigo-500));
+          width: rem(16);
+        }
+
+        &:hover {
+          background-color: var(--fl-fgColor, var(--indigo-800)) !important;
```

Comment:
Why are `!important` used here and at other places? Please avoid it whenever possible.